src/frontend/web_application/src/scenes/DiscussionList/components/ParticipantsIcon/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+import ParticipantIconLetter from '../../../../components/ParticipantIconLetter';
+import Icon from '../../../../components/Icon';
+import './style.scss';
+
+const ParticipantsIcon = ({ discussion }) => {
```

Comment:
This component is a good candidate for `componentShouldUpdate()` with re-render rules according to the number of participants